src/main/kotlin/no/nav/familie/ef/personhendelse/inntekt/VedtakendringerService.kt

```diff
@@ -96,4 +85,55 @@ class VedtakendringerService(
                 it.beskrivelse != "overgangsstoenadTilEnsligMorEllerFarSomBegynteAaLoepe1April2014EllerSenere"
         }?.groupBy { it.beskrivelse }?.size ?: 0
     }
+
+    private fun hentInntektshistorikk(fnr: String): InntektshistorikkResponse? {
+        try {
+            return inntektClient.hentInntektshistorikk(
+                fnr,
+                YearMonth.now().minusYears(1),
+                null
+            )
+        } catch (e: Exception) {
+            secureLogger.warn("Feil ved kall mot inntektskomponenten ved kall mot person $fnr. Message: ${e.message} Cause: ${e.cause}")
+        }
+        return null
+    }
+
+    private fun opprettOppgave(harNyeVedtak: Boolean, harEndretInntekt: Boolean, vedtakhendelseInntektberegning: VedtakhendelseInntektberegning) {
+        val oppgavetekst = lagOppgavetekst(harNyeVedtak, harEndretInntekt)
+        secureLogger.info("${vedtakhendelseInntektberegning.personIdent} - $oppgavetekst")
+        val oppgaveId = oppgaveClient.opprettOppgave(
+            OpprettOppgaveRequest(
+                ident = OppgaveIdentV2(ident = vedtakhendelseInntektberegning.personIdent, gruppe = IdentGruppe.FOLKEREGISTERIDENT),
+                saksId = null,
+                tema = Tema.ENF,
+                oppgavetype = Oppgavetype.VurderKonsekvensForYtelse,
+                fristFerdigstillelse = fristFerdigstillelse(),
+                beskrivelse = oppgavetekst,
+                enhetsnummer = null,
```

Comment:
Enhet er litt tricky;
Oppgave vil prøve å finne enhet for personen
I vår løsning setter vi enhet basert på relasjoner (integrasjoner: `api/arbeidsfordeling/enhet/{tema}/med-relasjoner`) 

I tillegg har vi ikke optimalt oppsett i norg2 så dnr-gjengen vil feile - med mindre vi setter 4489 manuelt hvis vi får `null` fra arbeidsfordelingen :)